lib/index.js

```diff
@@ -22,6 +22,12 @@ var DEFAULT_DELEGATE = {
     fs.mkdirSync(outputPath);
   },
   change: function(inputPath, outputPath, relativePath) {
+    // We no-op if the platform can symlink, because we assume the output path
+    // is already linked via a prior create operation.
+    if (util.canSymlink) {
+      return;
+    }
+
     fs.unlinkSync(outputPath);
```

Comment:
@trentmwillis yep, but i think the only thing we need to decide here is whether we explicitly support it or whether this is undefined behaviour territory.

My vote is it's undefined behaviour but feels like we should get some input from @stefanpenner as he brought up the concern.